### PR TITLE
pbTests: Change Fast-mode to change depending on other parameters

### PR DESCRIPTION
**Pull request by Willsparker**
As this script is mainly for the `vagrantPlaybookCheck` Jenkins job, I've only catered for the parameters that are able to be entered in that Jenkins job. (i.e. put in the cases for JDK8, 11, 13 and 14)
Fast-mode now gives the option for some roles to be skipped (mainly in the case of Windows), depending on what JDK is being built and it's variant, so that it only installs what's required to build that specific JDK.

ref: https://github.com/AdoptOpenJDK/openjdk-infrastructure/pull/1273#pullrequestreview-392833354

When given a chance, I'll put this through the VagrantPlaybookCheck job to ensure it works- I'll keep the [wip] tag until I can get a successful build on at least JDK8 and JDK11

### CODE REVIEW

File: ansible/pbTestScripts/vagrantPlaybookCheck.sh [hunk 1]
```diff
@@ -105,6 +106,20 @@ checkVars()
...
+	if [[ "$fastMode" == true ]]; then
+		skipFullSetup=",nvidia_cuda_toolkit"
+		case "$jdkToBuild" in
+			"jdk8" )
+				skipFullSetup="$skipFullSetup,MSVS_2017";
+				if [ "$buildHotspot" != "" ]; then
+					skipFullSetup="$skipFullSetup,MSVS_2010,VS2010_SP1"
+				fi
+				;;
+                	"jdk11" | "jdk13" | "jdk14" )
```
Comment (sxa) [inline ansible/pbTestScripts/vagrantPlaybookCheck.sh:118]: Please use a default `*)` section here instead of listing all versions other than jdk

**Comment by Willsparker** [comment]
https://ci.adoptopenjdk.net/view/Tooling/job/VagrantPlaybookCheck/OS=Win2012,label=infra-vagrant-1/485/console

Testing on JDK11

**Comment by Willsparker** [comment]
Rerun with the new conditions: https://ci.adoptopenjdk.net/view/Tooling/job/VagrantPlaybookCheck/486/

File: ansible/pbTestScripts/vagrantPlaybookCheck.sh (hunk 1 above)
Comment (Willsparker) [reply ansible/pbTestScripts/vagrantPlaybookCheck.sh]: Changed 👍

**Comment by Willsparker** [comment]
Note: We won't be able to properly test JDK8 until https://github.com/AdoptOpenJDK/openjdk-infrastructure/pull/1263 is in, as it currently won't start building JDK8 properly.

**Comment by Willsparker** [comment]
As the above PR is merged, I'm putting this PR though VPC:
https://ci.adoptopenjdk.net/view/Tooling/job/VagrantPlaybookCheck/517/
https://ci.adoptopenjdk.net/view/Tooling/job/VagrantPlaybookCheck/518/

The above are runs of JDK8 and JDK11 respectively. I've enabled the build but disabled tests as the test phase is relatively unstable currently, and the PR should only affect the build pre-reqs.

**Comment by Willsparker** [comment]
With the success of JDK8 's `VPC` run (https://ci.adoptopenjdk.net/view/Tooling/job/VagrantPlaybookCheck/519/)
And multiple successes of the JDK11 `VPC` run (https://ci.adoptopenjdk.net/view/Tooling/job/VagrantPlaybookCheck/521/
https://ci.adoptopenjdk.net/view/Tooling/job/VagrantPlaybookCheck/522/ )
This is ready for review :)

**Pull request merged**